webpack.config.js

```diff
@@ -1,32 +1,76 @@
 const Path = require('path');
```

Comment:
I would recommend creating separate webpack configs for dev/prod. Before we did our big restructure changes the apps webpack setup had three files:

A base config: contains the generic stuff like babel, entry/output points, css loader, etc.
A dev config which extends the base config: contains stuff like `watch: true` (not needed in prod) and source maps
A prod config which extends the base config: contains stuff like minification (that's the main thing)

`webpack-merge` is a great tool for this
https://github.com/survivejs/webpack-merge